core/embed/rust/src/ui/model_tr/ui_features_fw.rs

```diff
@@ -335,6 +335,36 @@ impl UIFeaturesFirmware for ModelTRFeatures {
         content_in_button_page(title, formatted, button, Some("".into()), false)
     }
 
+    fn confirm_value(
+        title: TString<'static>,
+        value: Obj,
+        description: Option<TString<'static>>,
+        _subtitle: Option<TString<'static>>,
+        verb: Option<TString<'static>>,
+        _verb_info: Option<TString<'static>>,
+        verb_cancel: Option<TString<'static>>,
+        _info_button: bool,
+        hold: bool,
+        _chunkify: bool,
+        _text_mono: bool,
+    ) -> Result<Gc<LayoutObj>, Error> {
+        let value: TString = value.try_into()?;
+        let description = description.unwrap_or("".into());
+        let paragraphs = Paragraphs::new([
+            Paragraph::new(&theme::TEXT_BOLD, description),
+            Paragraph::new(&theme::TEXT_MONO, value),
+        ]);
+
+        let layout = content_in_button_page(
+            title,
+            paragraphs,
+            verb.unwrap_or(TR::buttons__confirm.into()),
+            verb_cancel,
```

Comment:
Cancel button disappeared: https://data.trezor.io/dev/firmware/ui_report/12198553304/T3B1-en-core_device_test/diff/T3B1_en-device_tests-bitcoin-test_signtx.py%3A%3Atest_lock_time_blockheight.html

Reply:
Also for Update Transaction: https://data.trezor.io/dev/firmware/ui_report/12198553304/T3B1-en-core_device_test/diff/T3B1_en-device_tests-bitcoin-test_signtx_replacement.py::test_p2wpkh_in_p2sh_remove_change.html

Reply:
https://data.trezor.io/dev/firmware/ui_report/12198553304/T3B1-en-core_device_test/diff/T3B1_en-device_tests-solana-test_sign_tx.py::test_solana_sign_tx[transfer_token_with_compute_budget_without_heap-9ae0985e.html

Reply:
All these cases fixed should be fixed:
daa6bdf467dfe2763ea5c870bca9cb70138ac376